community-membership.md

```diff
@@ -1,77 +1,199 @@
+  - Focus on holistic acceptance of contribution such as dependencies with other features, backwards / forwards compatibility, API and flag definitions, etc
+- Expected to be responsive to review requests as per [community expectations](contributors/devel/community-expectations.md);
+- Mentor contributors and reviewers
+- May approve code contributions for acceptance
+
+## Area/Component owner
+
+Area / component areas can accept design proposals and approve design decisions for a part of the project.  This
+step may gate PRs for large pieces of functionality.
+
+### Owner requirements
+
+The following apply to the area / component for which one would be an owner.
+
+- Originally authored or contributed major functionality to an area
+- An approver in the **top-level** [OWNERS](contributors/devel/owners.md) files
+
+### Responsibilities and privileges
+
+The following apply to the area / component for which one would be an owner.
+
+- Owner status may be a precondition to accepting a new component or piece of major functionality
+- Design/proposal approval authority over the area / component, though escalation to kubernetes-maintainers is possible.
+- Mentor and guide approvers, reviewers, and members.
+  
+## Project maintainer
+
+[**Kubernetes project maintainers**](https://github.com/orgs/kubernetes/teams/kubernetes-maintainers) work holistically
+across the project to maintain its health and success.  They are typically involved in multiple different areas, and
+have made sustantial contributions both through code and broader organizational efforts.
+
+### Maintainer requirements
+
+- Approver for some part of the codebase for at least 3 months
+- Member for at least 1 year
+- Primary reviewer for 20 substantial PRs
+- Reviewed or merged at least 50 PRs
+- Apply to `kubernetes-maintainers@googlegroups.com`, with:
+  - A [Champion](https://github.com/kubernetes/community/blob/master/incubator.md#faq) from the existing
+    kubernetes-maintainers members
+  - A Sponsor from Project Approvers
+  - Summary of contributions to the project
+  - Current project responsibilities
+  - Links to merged and assigned PRs
+- At least 3 of the maintainers must approve the application, with no objections
+- Application expires after 2 weeks if not enough approvals are granted
+
+### Responsibilities and privileges
+
+- Write access to repo (assign issues/PRs, add/remove labels and milestones, edit issues and PRs, edit wiki, create/delete labels and milestones)
+- Capable of directly applying lgtm + approve labels for any PR, causing it to be merged by the submit queue - but permitted only in rare instances
+   - Expected to respect OWNERS files approvals and use standard procedure for merging code
+- Expected to work to holistically maintain the health of the project through:
+  - Reviewing PRs
+  - Fixing bugs
+  - Providing user support
+  - Mentoring and guiding approvers, reviewers, and other contributors
+
+## Project Approvers
```

Comment:
TL;DR

- **approvers**: responsible for initiatives within an area
- **owners:** responsible for entire area
- **leads:** responsible for all areas

We could probably collapse approvers - owners if we are trying to remove layers.

I believe "owners" are very close to "approvers", with the distinction that owners have more strategic responsibilities (approving feature proposal / designs, building out processes for managing the area, etc), whereas approvers tend to focus on specific initiatives within the component.

The leads role cuts across the organization.  Leads can act as owners for all areas / components.